content/en/docs/concepts/services-networking/endpoint-slices.md

```diff
@@ -24,15 +24,30 @@ Endpoints.
 
 {{% capture body %}}
 
+## Motivation
+
+The Endpoints API has provided a simple and straightforward way of
+tracking network endpoints in Kubernetes. Unfortunately as Kubernetes clusters
+and Services have gotten larger, limitations of that API became more visible.
+Most notably, those included challenges with scaling to larger numbers of
+network endpoints.
+
+Since all network endpoints for a Service were stored in a single Endpoints
+resource, those resources could get quite large. That affected the performance
+of Kubernetes components (notably the master control plane) and resulted in
+significant amounts of network traffic and processing when Endpoints changed.
+Endpoint Slices help you mitigate those issues as well as provide an extensible
+platform for additional features such as topological routing.
+
 ## Endpoint Slice resources {#endpointslice-resource}
 
-In Kubernetes, an EndpointSlice contains references to a set of network
-endpoints. The EndpointSlice controller automatically creates Endpoint Slices
+In Kubernetes, an Endpoint Slice contains references to a set of network
```

Comment:
I prefer `EndpointSlice`, see [documentation style guide's standards](https://kubernetes.io/docs/contribute/style/style-guide/#documentation-formatting-standards).

Reply:
Thanks for pointing that out! I'd missed that in the style guide, fixed now.